.github/workflows/appimage.yml

```diff
@@ -52,37 +60,29 @@ jobs:
       run: |
         sudo -u testbot bash -lc 'make setup'
 
-    - name: make build
```

Comment:
Confirming this was removed because it's already a dependency of "make appimage"?

Reply:
Correct!